mesonbuild/interpreter/primitives/string.py

```diff
@@ -118,11 +119,13 @@ def split_method(self, args: T.Tuple[T.Optional[str]], kwargs: TYPE_kwargs) -> T
         return self.held_object.split(args[0])
 
     @noKwargs
+    @FeatureNew('str.strip', '0.43.0')
```

Comment:
This was actually added in commit 5e2278d6d4e2337c05b2f4e583dc8620b90e9add with version 0.5.0, however in version 0.43.0 it began to accept optional posargs.

Please remove the FeatureNew. Unlike typed_kwargs, typed_pos_args does not directly support "since" checks. So you'll need to manually code it as an if block inside the function definition.

Reply:
Ok aded single use, let me know.